docs/reference/query-languages/esql/esql-enrich-data.md

```diff
@@ -15,6 +15,13 @@ For example, you can use `ENRICH` to:
 * Add product information to retail orders based on product IDs
 * Supplement contact information based on an email address
 
+[`ENRICH`](/reference/query-languages/esql/esql-commands.md#esql-enrich) is similar to [`LOOKUP join`](/reference/query-languages/esql/esql-commands.md#esql-lookup-join) in the fact that they both help you join data together. You should use `ENRICH` when:
+
+* Enrichment data doesn't changes frequently
+* You can accept index-time overhead
+* You are working with structured enrichment patterns
+* You can accept having multiple matches combined into multi-values
+* You can accept being limited to predefined match fields
```

Comment:
One more point: ENRICH's security model is very simplified; there is no restricting to specific enrich policies, no document and no field level security (I _think_). All of the above work for LOOKUP JOIN, however.

Reply:
I have added this. please let me know if this is now accurate